<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAddExistAccountClass</class>
 <widget class="QDialog" name="DialogAddExistAccountClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DialogAddExistAccount</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
      <item row="0" column="0">
       <widget class="QLabel" name="label_platform">
        <property name="text">
         <string>账户所属软件或平台名：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="ledit_platform"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_account">
        <property name="text">
         <string>账号：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="ledit_account"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_password">
        <property name="text">
         <string>密码：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="ledit_password"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pbtn_ok">
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbtn_cancel">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DialogAddExistAccount.qrc"/>
 </resources>
 <connections/>
</ui>
